SU2_CFD/src/fluid/CSU2TCLib.cpp

```diff
@@ -1598,6 +1633,8 @@ vector<su2double>& CSU2TCLib::GetDiffusionCoeff(){
    DiffusionCoeffWBE();
   if(Kind_TransCoeffModel == TRANSCOEFFMODEL::GUPTAYOS)
    DiffusionCoeffGY();
+  if(Kind_TransCoeffModel == TRANSCOEFFMODEL::SUTHERLAND)
+   DiffusionCoeffWBE();
```

Comment:
The diffusion coefficient used in the Sutherland model is still computed using the WBE routine